roles/etcd/tasks/certificates/fetch_server_certificates_from_ca.yml

```diff
@@ -3,9 +3,7 @@
   package:
     name: "etcd{{ '-' + etcd_version if etcd_version is defined else '' }}"
     state: present
-  when: not etcd_is_atomic | bool
```

Comment:
We can't install packages if etcd is on an atomic host.

I guess I really don't understand what the current model is supposed to look like still.

Reply:
Oh, correct, `etcd_is_atomic` check should remain.

If etcd being setup as a static pod (detected by `openshift_is_bootstrapped`) there is no need to install etcd from RPM - in this case on CentOS/RHEL there'd be no `etcd` user, so file ownership shouldn't be changed to `etcd`. In other cases either an etcd image is used (on Atomic hosts) or an RPM is being installed (on dedicated etcd RHEL hosts)